project/Dependencies.scala

```diff
@@ -72,9 +72,9 @@ object Dependencies {
   val pureconfig          = "com.github.pureconfig"      %% "pureconfig"                % "0.14.0"
   val scalaLogging        = "com.typesafe.scala-logging" %% "scala-logging"             % "3.9.4"
   val scalaUri            = "io.lemonlabs"               %% "scala-uri"                 % "3.0.0"
-  val scalacheck          = "org.scalacheck"             %% "scalacheck"                % "1.15.2"
-  val scalacheckShapeless = "com.github.alexarchambault" %% "scalacheck-shapeless_1.14" % "1.2.5" % "test"
-  val scalactic           = "org.scalactic"              %% "scalactic"                 % "3.0.5" % "test"
+  val scalacheck          = "org.scalacheck"             %% "scalacheck"                % "1.15.0"
+  val scalacheckShapeless = "com.github.alexarchambault" %% "scalacheck-shapeless_1.14" % "1.2.5"  % "test"
```

Comment:
Should _scalacheck-shapeless_ be updated to `scalacheck-shapeless_1.15` as well?

Reply:
Yes, you are right. On the [releases](https://github.com/alexarchambault/scalacheck-shapeless/releases) page I see that scalacheck 1.15 used from 1.3.0 version (the latest release for now). I [upgraded](https://github.com/rchain/rchain/pull/3721/commits/796307f35078eb50c2977db7a5a5a1c9ba456d9e) scalacheck-shapeless version.